internal/manifests/internal/gateway/gateway-tenants.yaml

```diff
@@ -1,9 +1,109 @@
+- name: {{ $spec.Name }}
+  id: {{ $spec.ID }}
+  oidc:
+    {{- range $secret := $l.TenantSecrets }}
+    {{- if eq $secret.TenantName $spec.Name -}}
+    {{ if $secret.ClientID }}
+    clientID: {{ $secret.ClientID }}
+    {{- end -}}
+    {{ if $secret.ClientSecret }}
+    clientSecret: {{ $secret.ClientSecret }}
+    {{- end -}}
+    {{ if $secret.IssuerCAPath }}
+    issuerCAPath: {{ $secret.IssuerCAPath }}
+    {{- end -}}
+    {{- end -}}
+    {{- end -}}
+    {{ print "\n" }}
+    issuerURL: {{ $spec.OIDC.IssuerURL }}
+    redirectURL: {{ $spec.OIDC.RedirectURL }}
+    {{ if $spec.OIDC.UsernameClaim }}
+    usernameClaim: {{ $spec.OIDC.UsernameClaim }}
+    {{- end -}}
+    {{ if $spec.OIDC.GroupClaim }}
+    groupClaim: {{ $spec.OIDC.GroupClaim }}
+    {{- end -}}
+  {{ print "\n" }}
+  opa:
+    query: data.lokistack.allow
+    paths:
+    - /etc/lokistack-gateway/rbac.yaml
+    - /etc/lokistack-gateway/lokistack-gateway.rego
+{{- end -}}
+{{- else if eq $l.Stack.Tenants.Mode "dynamic" -}}
+{{- if $tenant := $l.Stack.Tenants -}}
+{{- range $spec := $tenant.Authentication }}
+- name: {{ $spec.Name }}
+  id: {{ $spec.ID }}
+  oidc:
+    {{- range $secret := $l.TenantSecrets }}
+    {{- if eq $secret.TenantName $spec.Name -}}
+    {{ if $secret.ClientID }}
+    clientID: {{ $secret.ClientID }}
+    {{- end -}}
+    {{ if $secret.ClientID }}
+    clientSecret: {{ $secret.ClientSecret }}
+    {{- end -}}
+    {{ if $secret.ClientID }}
+    issuerCAPath: {{ $secret.IssuerCAPath }}
+    {{- end -}}
+    {{- end -}}
+    {{- end -}}
+    {{ print "\n" }}
+    issuerURL: {{ $spec.OIDC.IssuerURL }}
+    redirectURL: {{ $spec.OIDC.RedirectURL }}
+    {{ if $spec.OIDC.UsernameClaim }}
+    usernameClaim: {{ $spec.OIDC.UsernameClaim }}
+    {{- end -}}
+    {{ if $spec.OIDC.GroupClaim }}
+    groupClaim: {{ $spec.OIDC.GroupClaim }}
+    {{- end -}}
+  {{ print "\n" }}
+  opa:
+    url: {{ $tenant.Authorization.OPA.URL }}
+{{- end -}}
+{{- end -}}
+{{- else -}}
+{{ print "\n" }}
+- name: application
+  id: 32e45e3e-b760-43a2-a7e1-02c5631e56e9
+  oidc:
+    clientID: test
+    # TODO - these need to be replaced once we integrate dex into lokistack-gateway.
+    clientSecret: ZXhhbXBsZS1hcHAtc2VjcmV0
+    issuerCAPath: ./tmp/certs/ca.pem
+    issuerURL: https://127.0.0.1:5556/dex
+    redirectURL: https://localhost:8443/oidc/application/callback
+    usernameClaim: name
+  opa:
+    url: http://127.0.0.1:8080/v1/data/observatorium/allow
+- name: infrastructure
+  id: 40de0532-10a2-430c-9a00-62c46455c118
+  oidc:
+    clientID: test
+    # TODO - these need to be replaced once we integrate dex into lokistack-gateway.
+    clientSecret: ZXhhbXBsZS1hcHAtc2VjcmV0
+    issuerCAPath: ./tmp/certs/ca.pem
+    issuerURL: https://127.0.0.1:5556/dex
+    redirectURL: https://localhost:8443/oidc/infrastructure/callback
+    usernameClaim: name
+  opa:
+    url: http://127.0.0.1:8080/v1/data/observatorium/allow
```

Comment:
This URL is invalid since we set the `package` in the rego file to `lokistack`. The correct URL is:
`http://127.0.0.1:8080/v1/data/lokistack/allow`